### PR TITLE
[SYCL] Apply post-commit comments for the documentation

### PR DESCRIPTION
**Pull request by vladimirlaz**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,10 +2,9 @@
 
 ## License
 
-Intel Project for LLVM\* technology is licensed under the terms of the Apache
-License v2.0 with LLVM Exceptions license ([LICENSE.txt](llvm/LICENSE.TXT)) to
-ensure our ability to contribute this project to the LLVM project under the
-same license.
+This project is licensed under the terms of the Apache License v2.0 with LLVM
+Exceptions license ([LICENSE.txt](llvm/LICENSE.TXT)) to ensure our ability to
```
Comment (pvchupin) [inline CONTRIBUTING.md]: Broken link. Please update to correct upstream license.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+LLVM test suite Repository
```
Comment (pvchupin) [inline README.md]: Should it be header in md format?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+See [LICENSE.txt](LICENSE.TXT) for details.
+
+## Contributing
+
+See [CONTRIBUTING.md](CONTRIBUTING.md) for details.
+
+## Sub-projects Documentation
+
+* oneAPI Data Parallel C++ compiler - See
+  [DPC++ Documentation](https://intel.github.io/llvm-docs/)
```
Comment (pvchupin) [inline README.md]: This is not subproject of llvm-test-suite project. You can refer as some related project.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+\*Other names and brands may be claimed as the property of others.
```
Comment (pvchupin) [inline README.md:27]: There should be stars added above to "LLVM".

**Review by pvchupin — CHANGES_REQUESTED** [review]

**Comment by vladimirlaz** [comment]
@pvchupin, @andykaylor  could you please have a look and approve/comment?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -2,10 +2,9 @@
 
 ## License
 
-Intel Project for LLVM\* technology is licensed under the terms of the Apache
-License v2.0 with LLVM Exceptions license ([LICENSE.txt](llvm/LICENSE.TXT)) to
-ensure our ability to contribute this project to the LLVM project under the
-same license.
+This project is licensed under the terms of the Apache License v2.0 with LLVM*
```
Comment (pvchupin) [inline CONTRIBUTING.md]:
```suggestion
This project is licensed under the terms of the Apache License v2.0 with LLVM
```

**Review by pvchupin — DISMISSED** [review]
One edit. Otherwise look good.

**Pull request merged**